Libplanet/Net/IceServer.cs

```diff
@@ -42,30 +43,27 @@ public class IceServer
                 {
                     if (url.Scheme != "turn")
                     {
-                        throw new ArgumentException($"{url} isn't valid TURN url.");
+                        throw new ArgumentException($"{url} is not a valid TURN url.");
                     }
 
                     int port = url.IsDefaultPort
                         ? TurnClient.TurnDefaultPort
                         : url.Port;
-                    var turnClient = new TurnClient(
+                    TurnClient turnClient = new TurnClient(
```

Comment:
It's better to use `var` here as the type is apparent.  See also: <https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#implicitly-typed-local-variables>.

Reply:
Eh, I'm still on the fence about it, but the argument is fair enough. Fixed; see e7d5dfdab0e30cb9d4ea9e06edd972e1641ad705.